usaspending_api/etl/transaction_loaders/fpds_loader.py

```diff
@@ -0,0 +1,289 @@
+import psycopg2
+import logging
+from os import environ
+
+from usaspending_api.etl.transaction_loaders.field_mappings_fpds import (
+    transaction_fpds_columns,
+    transaction_normalized_columns,
+    transaction_normalized_functions,
+    legal_entity_columns,
+    legal_entity_boolean_columns,
+    legal_entity_functions,
+    recipient_location_columns,
+    recipient_location_functions,
+    place_of_performance_columns,
+    place_of_performance_functions,
+    award_functions,
+    transaction_fpds_boolean_columns,
+    transaction_fpds_functions,
+)
+from usaspending_api.etl.transaction_loaders.data_load_helpers import (
+    capitalize_if_string,
+    false_if_null,
+    get_deleted_fpds_data_from_s3,
+)
+from usaspending_api.etl.transaction_loaders.generic_loaders import (
+    update_transaction_fpds,
+    update_transaction_normalized,
+    insert_transaction_normalized,
+    insert_transaction_fpds,
+    bulk_insert_recipient_location,
+    bulk_insert_recipient,
+    bulk_insert_place_of_performance,
+    insert_award,
+)
+from usaspending_api.common.helpers.timing_helpers import Timer
+from usaspending_api.common.helpers.sql_helpers import get_database_dsn_string
+
+
+USASPENDING_CONNECTION_STRING = get_database_dsn_string()
+BROKER_CONNECTION_STRING = environ.get("DATA_BROKER_DATABASE_URL", None)
```

Comment:
Still need to remove direct env var access

Reply:
I was hoping to wait and just merge it https://github.com/fedspendingtransparency/usaspending-api/pull/2058/files first, since the work is conflicting. Not sure how long that change is going to take now...

Reply:
Keith was actively developing last night, most of the bugs I noticed have been resolved

Reply:
At this point, it would be better to make the necessary changes to `pytest_configure()` in conftest.py and deal with any merge conflict later.

Reply:
my pleasure...